### PR TITLE
adding templates

### PR DESCRIPTION
**Pull request by krnese**
templates for adding storage accounts to OMS workspace

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@krnese__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (krnese). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by krnese** [comment]
I have tested and verified this template manually, as it rely on existing RG and storage account

File: oms-storageaccount/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,109 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "omsworkspaceName": {
```
Comment (singhkays) [inline oms-storageaccount/azuredeploy.json]: camelCase

File: oms-storageaccount/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+      "allowedValues": [
+        "West Europe",
+        "East US",
+        "Southeast Asia"
+      ],
+      "metadata": {
+        "description": "Select OMS region"
+      }
+    },
+    "storageaccountName": {
```
Comment (singhkays) [inline oms-storageaccount/azuredeploy.json]: camelCase

File: oms-storageaccount/azuredeploy.json [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+      "type": "string",
+      "defaultValue": "mystorage",
+      "metadata": {
+        "description": "Specify the name of the existing storage account to add to OMS"
+      }
+    },
+    "storageaccountresourcegroupName": {
```
Comment (singhkays) [inline oms-storageaccount/azuredeploy.json]: prefix with "existing"

**Comment by krnese** [comment]
tested and verified.

**Comment by azurecla** [comment]
Hi __@krnese__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (krnese). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by krnese** [comment]
tested and verified

**Comment by singhkays** [comment]
lgtm, merged

**Pull request merged**